### ISSUE TITLE
iter

### ISSUE THREAD

**Issue by dzmitry-lahoda** [cloudhead/nonempty#30]
for vec next works
```
 &command.document_ids
```

for non empty asked for
```
 &command.document_ids.iter() 
```

seems not all rust stuff implemented in that area

**Comment by FintanH** [comment]
Just to clarify, this is when using a `for` loop, right?

For example,
```rust
for id in & command.document_ids {
  /* etc. */
}
```

**Comment by cloudhead** [comment]
Fixed in 7496a26f373baff7ef8ef82d7fbaf7ef655add25
